processor/src/operations/crypto_ops.rs

```diff
@@ -1,6 +1,6 @@
 use vm_core::StarkField;
 
-use super::{ExecutionError, Operation, Process};
+use super::{AdviceProvider, ExecutionError, Operation, Process};
```

Comment:
minor nit: I'd probably get rid of the blank line between these two imports.

Reply:
Done!